### PR TITLE
add globvars and warning of two possible backends

### PR DESCRIPTION
**Pull request by TomorrowIsAnOtherDay**

### CODE REVIEW

**Comment by TomorrowIsAnOtherDay** [comment]
fix #998

File: parl/utils/globvars.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import types
+CONFIG = types.SimpleNamespace
+
+class GlobalConfig(CONFIG):
+    """
+    A namespace to store global variables.
+    Example:
+    .. code-block:: none
+        from parl.utils.globvars import global_config as config
```
Comment (rical730) [inline parl/utils/globvars.py]: It can be replaced by `from parl.utils import ...` after setting in parl/utils/__init__.py

Comment (TomorrowIsAnOtherDay) [reply parl/utils/globvars.py]: done.

**Review by rical730 — APPROVED** [review]
LGTM~

**Pull request merged**